tc-health-client/tmagent/tmagent.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/tc-health-client/config"
 	"github.com/apache/trafficcontrol/tc-health-client/util"
-	"github.com/apache/trafficcontrol/traffic_monitor/datareq"
+	//"github.com/apache/trafficcontrol/traffic_monitor/datareq"
```

Comment:
This comment should be removed:

Reply:
@zrhoffman yeah, agreed.  I overlooked that.